test/unit/models/location_error_test.rb

```diff
@@ -5,22 +5,6 @@ class LocationErrorTest < ActiveSupport::TestCase
       assert_equal(error.message, "formats.local_transaction.invalid_postcode")
     end
 
-    context "when given a postcode_error" do
-      should "send the postcode error as a notification" do
-        ActiveSupport::Notifications.expects(:instrument).with("postcode_error_notification", postcode_error: "some_error")
-
-        LocationError.new("some_error")
-      end
-    end
-
-    context "when not given a postcode_error" do
-      should "not send a postcode_error notification" do
-        ActiveSupport::Notifications.expects(:instrument).never
-
-        LocationError.new
-      end
-    end
```

Comment:
I noticed that the `"fullPostcodeNoMapitMatch"` case isn't tested in here, which I guess isn't related to these changes as it wasn't tested before... but could be something we add separately?

Reply:
Thanks for pointing out. I've refactored this code a bit so every branch doesn't need testing. It was getting rather closer to adding unit tests to test configuration.

Reply:
👍